app/src/main/java/org/oppia/app/recyclerview/BindableAdapter.kt

```diff
@@ -64,7 +65,7 @@ class BindableAdapter<T : Any> internal constructor(
           "expected adapter class type: $dataClassType."
       }
     }
-    @Suppress("UNCHECKED_CAST") // This is safe. See the above check.
```

Comment:
Why remove this comment?

Reply:
Oh, this is my mistake. Will put it again in next commit.

Reply:
Resolving this on behalf of ben